lib/widgets/autocomplete.dart

```diff
@@ -195,24 +198,56 @@ class ComposeAutocomplete extends AutocompleteField<MentionAutocompleteQuery, Me
 
   @override
   Widget buildItem(BuildContext context, int index, MentionAutocompleteResult option) {
+    final designVariables = DesignVariables.of(context);
     Widget avatar;
     String label;
+    String? metadata;
+
     switch (option) {
       case UserMentionAutocompleteResult(:var userId):
-        avatar = Avatar(userId: userId, size: 32, borderRadius: 3);
-        label = PerAccountStoreWidget.of(context).users[userId]!.fullName;
+        final store = PerAccountStoreWidget.of(context);
+        final user = store.users[userId]!;
+        avatar = Avatar(userId: userId, size: 36, borderRadius: 4);
+        label = user.fullName;
+        metadata = getDisplayEmailFor(user, store: store);
```

Comment:
"metadata" isn't really a good term for this (though I see it appears in the Figma design). It means "data about data"; but a user's email address is data about the user, just as much as their name is.

Ultimately what this variable is for — currently it's just email address but the design suggests including pronouns, and probably it'll include group descriptions and so on for other types of results — is just that it's a string that appears alongside the label, with less emphasis than the label.

So a good term would be "sublabel". Possible other directions would include "extra", "extraLabel", "secondLine".